ray-on-gke/user/kuberay-tpu-webhook/main.go

```diff
@@ -0,0 +1,242 @@
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"strings"
+
+	ray "github.com/ray-project/kuberay/ray-operator/apis/ray/v1"
```

Comment:
Does this mean we only support v1 RayClusters?

Reply:
The webhook should work with both v1 and v1alpha1, my intuition was just to import the stable release.